<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMI</class>
 <widget class="QDialog" name="BMI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人体BMI计算器</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_weight">
              <property name="text">
               <string>请输入你的体重(KG)：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_weight">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_height">
              <property name="text">
               <string>请输入你的身高(cm)：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_height"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>计算BMI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_suggest">
        <property name="text">
         <string>提示：1. 中国成人居民BMI衡量标准是小于等于18.1为偏瘦；18.5-23.9为正常；24-27.9为超重；大于等于28为肥胖。2. 最理想的BMI数值为22，理想体重 = 22 x 身高2 (米)，±10%波动</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>您的计算结果：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>BMI：27.18，属于超重 您的理想体重为：68KG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
